frontend/src/components/modals/AddTeamMemberModal.tsx

```diff
@@ -0,0 +1,142 @@
+import { RootState } from '../../redux/types';
+import { ModalTypes, Modal } from './types';
+import { LoadingSpinner } from '../LoadingSpinner';
+import { ModalContent } from './modalparts/ModalContent';
+import { ModalFooter } from './modalparts/ModalFooter';
+import { ModalFooterButtonDiv } from './modalparts/ModalFooterButtonDiv';
+import { ModalHeader } from './modalparts/ModalHeader';
+import { SelectMemberRole } from './modalparts/TeamMemberModalParts';
+import { Input } from '../forms/FormField';
+import css from './AddTeamMemberModal.module.scss';
+
+const classes = classNames.bind(css);
+
+export const AddTeamMemberModal: Modal<ModalTypes.ADD_TEAM_MEMBER_MODAL> = ({
+  closeModal,
+}) => {
+  const { t } = useTranslation();
+  const dispatch = useDispatch<StoreDispatchType>();
+  const [isLoading, setIsLoading] = useState(false);
+  const [errorMessage, setErrorMessage] = useState('');
+  const [openInfo, setOpenInfo] = useState(true);
+  const [formValues, setFormValues] = useState({
+    type: RoleType.Developer,
+    email: '',
+  });
+  const chosenRoadmapId = useSelector<RootState, number | undefined>(
+    chosenRoadmapIdSelector,
+  );
+
+  const handleSubmit = async (event: FormEvent<HTMLFormElement>) => {
+    event.preventDefault();
+    event.stopPropagation();
+
+    setIsLoading(true);
+    const res = await dispatch(
+      roadmapsActions.sendInvitation({
+        email: formValues.email,
+        type: formValues.type,
+        roadmapId: chosenRoadmapId!,
+      }),
+    );
+    setIsLoading(false);
+    if (roadmapsActions.sendInvitation.rejected.match(res)) {
+      if (res.payload?.message) setErrorMessage(res.payload.message);
+      return;
+    }
+    closeModal();
+  };
+
+  return (
+    <Form onSubmit={handleSubmit}>
+      <ModalHeader closeModal={closeModal}>
+        <h3>
+          <Trans i18nKey="Add a team member" />
+        </h3>
+      </ModalHeader>
+      <ModalContent>
+        <div className={classes(css.addMemberContent)}>
+          <div className={classes(css.instructions)}>
+            {openInfo ? (
+              <>
+                <b>
+                  <Trans i18nKey="Here’s how to add a team member" />
+                </b>{' '}
+                <Trans i18nKey="Team member addition instructions" />{' '}
+                <button
+                  className={classes(css.linkButton, css.blue)}
+                  tabIndex={0}
+                  type="button"
+                  onClick={() => setOpenInfo(false)}
+                >
+                  <Trans i18nKey="Hide info" />
+                </button>
+              </>
+            ) : (
+              <button
+                className={classes(css.linkButton, css.blue)}
+                tabIndex={0}
+                type="button"
+                onClick={() => setOpenInfo(true)}
+              >
+                <Trans i18nKey="Show info" />
+              </button>
+            )}
```

Comment:
This *might* be a bit nicer with conditionally showing the info text, and then the button conditionally selecting its text and toggling the state
```suggestion
            {openInfo && (
              <>
                <b>
                  <Trans i18nKey="Here’s how to add a team member" />
                </b>{' '}
                <Trans i18nKey="Team member addition instructions" />{' '}
              </>
            )}
            <button
              className={classes(css.linkButton, css.blue)}
              tabIndex={0}
              type="button"
              onClick={() => setOpenInfo((prev) => !prev)}
            >
              <Trans i18nKey={openInfo ? 'Hide info' : 'Show info'} />
            </button>
```